### ISSUE TITLE
Only First Line of the Original Table of a Flowfield is Shown

### ISSUE THREAD

**Issue by asbkevl** [hougaard/Simple-Object-Designer#356]
Hi Erik,

TL;DR: I wanted to transfer a flowfield from item card to sales line, able to compile but the result is not as intended.

I created a flowfield in Item Card (27) to show Description of Tariff Number (260).
This flowfield work as intended.
See Ref 1 and Ref 2.

Then, I transfer this flowfield to Sales Line using [Field Transfers] function.
The flowfield able to show but it's not coupled with Item No.
Seems like only the Description from the 1st line of Tariff Number is shown.
Is this a bug or limitation?
See Ref 3.

### Ref 1: Table 260 Tariff Numbers
![image](https://github.com/hougaard/Simple-Object-Designer/assets/143057361/650db891-ceb2-4405-a7b8-4dcc7150cc64)

### Ref 2: SOD Field Setting
![image](https://github.com/hougaard/Simple-Object-Designer/assets/143057361/76a8f091-a6ac-4822-bf20-a090637e0041)

### Ref 3: Showing Flowfield in Item List
![image](https://github.com/hougaard/Simple-Object-Designer/assets/143057361/e5faebc3-7ce0-46aa-84c9-dfbb10dcf773)

### Ref 3: Sales Line
![image](https://github.com/hougaard/Simple-Object-Designer/assets/143057361/3cadb378-00fb-42ee-90b3-0d3d3518d590)
